user/contacts.d.ts

```diff
@@ -6,6 +6,8 @@
 declare module 'libsession_util_nodejs' {
   type ContactsWrapper = BaseConfigWrapper & {
     init: (secretKey: Uint8Array, dump: Uint8Array | null) => void;
+    /** This function is used to free wrappers from memory only */
+    free: () => void;
     get: (pubkeyHex: string) => ContactInfo | null;
     set: (contact: ContactInfoSet) => void;
     getAll: () => Array<ContactInfo>;
```

Comment:
you didn't commit any cpp files update. Surely that's wrong.

Reply:
as mentioned above free() is only used in the frontend for wrappers that are stored in memory and haven't been saved to the database yet, there is no direct call to libsession but we want the typings